sysl2/sysl/sysl_test.go

```diff
@@ -609,12 +609,73 @@ func TestMain2WithBinaryInfoCmd(t *testing.T) {
 	assert.Equal(t, 0, exitCode)
 }
 
-func TestSwaggerExportCmd(t *testing.T) {
+func TestSwaggerAppExportCd(t *testing.T) {
 	t.Parallel()
 	logger, _ := test.NewNullLogger()
 	_, fs := testutil.WriteToMemOverlayFs(".")
-	main2([]string{"sysl", "export-swagger", "-o", "SIMPLE_SWAGGER_EXAMPLE.yaml", "-a", "testapp",
+	main2([]string{"sysl", "export", "-o", "SIMPLE_SWAGGER_EXAMPLE.yaml", "-a", "testapp",
 		"exporter/test-data/SIMPLE_SWAGGER_EXAMPLE.sysl"}, fs, logger, main3)
 	_, err := ioutil.ReadFile("SIMPLE_SWAGGER_EXAMPLE.yaml")
 	assert.NoError(t, err)
+	os.Remove("SIMPLE_SWAGGER_EXAMPLE.yaml")
```

Comment:
Please check the other tests that use WriteToMemOverlayFs. That fs implementation doesnt write anything to the real filesystem so this remove shouldnt be needed. I think you should be passing memfs to the main2() (where memfs is the _'ed return value)

Reply:
ok